app/lib/obituary_microservice.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../poros/nytimes_results.rb'
+require_relative '../serializers/obituary_serializer.rb'
+require 'jsonapi-serializers'
+
+
+class ObituaryMicroservice < Sinatra::Base
+  get '/' do
+    obituaries = NytimesResults.new.get_obituaries
+    ObituarySerializer.new.format_obituaries(obituaries).to_json
+  end
```

Comment:
This is our routes/controller for this app.

Reply:
those are some FIIINNNNNNNE class and variable names!

Reply:
![](https://media1.giphy.com/media/l49JBNqz3vcNJNubS/giphy.gif?cid=5a38a5a29aw67vg3ygn2n3auqwkwh51ha36xow9syyc5qcoe&rid=giphy.gif)